test/libsolidity/SyntaxTest.h

```diff
@@ -54,8 +54,8 @@ class SyntaxTest: AnalysisFramework, public TestCase
 {
 public:
 	static std::unique_ptr<TestCase> create(Config const& _config)
-	{ return std::unique_ptr<TestCase>(new SyntaxTest(_config.filename)); }
-	SyntaxTest(std::string const& _filename);
+	{ return std::unique_ptr<TestCase>(new SyntaxTest(_config.filename, _config.evmVersion)); }
+	SyntaxTest(std::string const& _filename, langutil::EVMVersion const _evmVersion);
```

Comment:
I know it's not *your* code change I'm suggesting to change here, but wouldn't it not hurt to change this `unique_ptr<T>(new ...)` into a `make_unique<T>(...)`? -- one of those lines we could easily change wile walking over them. :-)

Reply:
True - I didn't even look at it all that much and just inserted the argument down the call chain - but yeah, ``make_unique`` is of course much better, since we now have it - I'll change it!